doc/playbook.rst

```diff
@@ -89,6 +92,29 @@ To configure these notifications, in a shell:
 This creates a notification on the given topic using JSON format when an object has been finalized (transfer of it has completed).
 Notifications are only sent on this topic for objects with the instrument name as a prefix.
 
+USDF
+----
+
+The bucket ``rubin-pp`` holds incoming raw images.
+
+The bucket ``rubin-pp-users`` holds the central repository described in `DMTN-219`_.
+The central repository currently contains:
+
+* HSC: a copy of `ap_verify_ci_cosmos_pdr2/preloaded@u/kfindeisen/DM-35052-expansion <https://github.com/lsst/ap_verify_ci_cosmos_pdr2/tree/u/kfindeisen/DM-35052-expansion/preloaded>`_.
```

Comment:
Just to check since I am not familiar with this usage, the @ is intentional in the path, correct?

Reply:
Yeah, it's an adaptation of GutHub's [standard notation for commit hashes](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#commit-shas). I didn't know of a better way to indicate that it has to be that specific branch. Should I remove it?

Reply:
I'd leave it in since its important they knew its a specific branch. Everything worked correctly with the link, I just hadn't seen it like that before and figured it was something I was unfamiliar with.